src/VISABConnector.Unity/ImageCreator.cs

```diff
@@ -146,5 +143,67 @@ public static void SetLayerRecursively(GameObject obj, int layer)
                 SetLayerRecursively(child.gameObject, layer);
             }
         }
+
+        /// <summary>
+        /// Assigns layer to just child or parent (hence all children)
+        /// </summary>
+        /// <param name="obj"></param>
+        /// <param name="config"></param>
+        /// <param name="layer"></param>
```

Comment:
The parameters should have comments aswell.
Also, this method could be private right?